tests_integration/spcs/test_cp.py

```diff
@@ -20,7 +20,7 @@ def test_cp(runner, snowflake_session):
             "create",
             "--name",
             cp_name,
-            "--num",
+            "--min-nodes",
```

Comment:
I would appreciate you adding some integration test for services and compute pool.